src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -600,6 +617,45 @@ public void setDropTruncateTableEnabled(boolean enabled)
     }
 
     @Override
+<<<<<<< HEAD
+=======
+    public boolean getBulkLoadEnabled()
+    {
+        return DEFAULT_CONFIG.getBulkLoadEnabled();
+    }
+
+    @Override
+    public void setBulkLoadEnabled(boolean enabled)
+    {
+        DEFAULT_CONFIG.setBulkLoadEnabled(enabled);
+    }
+
+    @Override
+    public boolean getDDLEnabled()
+    {
+        return DEFAULT_CONFIG.getDDLEnabled();
+    }
+
+    @Override
+    public void setDDLEnabled(boolean enabled)
+    {
+        DEFAULT_CONFIG.setDDLEnabled(enabled);
+    }
+
+    @Override
+    public boolean getDCLEnabled()
+    {
+        return DEFAULT_CONFIG.getDCLEnabled();
+    }
+
+    @Override
+    public void setDCLEnabled(boolean enabled)
+    {
+        DEFAULT_CONFIG.setDCLEnabled(enabled);
+    }
+
+    @Override
+>>>>>>> 33ca0adba8 ((C4.1) disable ddl/dcl under guardrail framework)
```

Comment:
@Yukei7

Reply:
Sorry didn't cleanup the cherry-picking. updated